code/game/sound.dm

```diff
@@ -102,7 +102,7 @@
 	UNTIL(SSticker.login_music) //wait for SSticker init to set the login music
 
 	if(prefs && (prefs.toggles & SOUND_LOBBY))
-		SEND_SOUND(src, sound(SSticker.login_music, repeat = 0, wait = 0, volume = vol, channel = CHANNEL_LOBBYMUSIC) // MAD JAMS)
+		SEND_SOUND(src, sound(SSticker.login_music, repeat = 0, wait = 0, volume = vol, channel = CHANNEL_LOBBYMUSIC)) // MAD JAMS
```

Comment:
shouldn't this be erroring?

Reply:
No.
```
SEND_SOUND(atom, sound //comment)
->
DIRECT_OUTPUT(atom, sound //comment)
->
A << B //comment
```

But it is an abuse of the fact macros work like that.